libs/jquery-ui-datepicker-fr-FR/jquery-ui-datepicker-fr-FR.js

```diff
@@ -0,0 +1,100 @@
+/* French (fr-FR) translations for the jQuery UI date picker plugin. */
+
+jQuery(function ($) {
+    var langCode = 'fr-FR';
+
+    $.datepicker.regional[langCode] = {
+        // Strings
+        closeText: 'Fermer',
+        prevText: '&#x3c;' + 'Précédent',
+        nextText: 'Suivant' + '&#x3e;',
+        currentText: 'Aujourd\'hui',
+        monthNames: [
+            'Janvier',
+            'Février',
+            'Mars',
+            'Avril',
+            'Mai',
+            'Juin',
+            'Juillet',
+            'Août',
+            'Septembre',
+            'Octobre',
+            'Novembre',
+            'Décembre'
+        ],
+        monthNamesShort: [
+            'Jan',
+            'Fév',
+            'Mars',
+            'Avr',
+            'Mai',
+            'Juin',
+            'Juil',
+            'Août',
+            'Sep',
+            'Oct',
+            'Nov',
+            'Déc'
+        ],
+        dayNames: [
+            'Dimanche',
+            'Lundi',
+            'Mardi',
+            'Mercredi',
+            'Jeudi',
+            'Vendredi',
+            'Samedi'
+        ],
+        dayNamesShort: [
+            'Dim',
+            'Lun',
+            'Mar',
+            'Mer',
+            'Jeu',
+            'Ven',
+            'Sam'
+        ],
+        dayNamesMin: [
+            'Di',
+            'Lu',
+            'Ma',
+            'Me',
+            'Je',
+            'Ve',
+            'Sa'
+        ],
+        weekHeader: 'Sem',
+
+        // Settings
+        dateFormat: 'dd-mm-yy',
+        firstDay: 1,
+        isRTL: false,
+        showMonthAfterYear: false,
+        yearSuffix: ''
+    };
+
+    $.datepicker.setDefaults($.datepicker.regional[langCode]);
+
+    if (!($.timepicker)) {
+        return;
+    }
+
+    $.timepicker.regional[langCode] = {
+        // Strings
+        timeOnlyTitle: 'Select time',
```

Comment:
Looks like this string has not been translated. Same thing in the `.min.js` file.

Reply:
Correct. I have fixed that, and committed both files again.